### DISCUSSION TITLE
Listing Items API support

### DISCUSSION THREAD

**Discussion by igoryan-k** [abuzuhri/Amazon-SP-API-CSharp#36] [Q&A]
Hello! Thank you, this is a great library! 

I was testing product price updates and looks like feeds are very slow and fall under rate restrictions. 
Not sure how scalable is this approach. 

It looks like there is another way of updating prices via the new SP API by using Listings Items API patch requests  https://github.com/amzn/selling-partner-api-docs/blob/main/guides/en-US/use-case-guides/listings-items-api-use-case-guide/listings-items-api-use-case-guide_2020-09-01.md#tutorial-partially-update-a-listing

Are you planning on adding this functionality? I saw you had LIstingsItemService for this API. 
If not, I might be able to contribute by adding this functionality by end of this month

**Comment by abuzuhri** [comment]
Hello @igoryan-k ,

Thanks for using my Library

> I was testing product price updates and looks like feeds are very slow and fall under rate restrictions.
> Not sure how scalable is this approach.

I use Feed to update price and every patch i send 50k Update and i never face any problem with slow, do you know that you can send many updates in one feed ,  please give more information how you currenty use the feed for Update price .




> It looks like there is another way of updating prices via the new SP API by using Listings Items API patch requests https://github.com/amzn/selling-partner-api-docs/blob/main/guides/en-US/use-case-guides/listings-items-api-use-case-guide/listings-items-api-use-case-guide_2020-09-01.md#tutorial-partially-update-a-listing
> 
> Are you planning on adding this functionality? I saw you had LIstingsItemService for this API. If not, I might be able to contribute by adding this functionality by end of this month

Yes I'm Open to any update and fix in the library , I add it as task and going to work in it this weekend or next weekend , if you free in Christmas and you can do that then I will be big thanks to you . ListingsItemService is empty now but you can start use it :)

**Reply by igoryan-k** [reply]
Hi @abuzuhri , thanks for your reply!
Sure, I am going to try to send multiple price updates and see how it goes. 
I will be free week of 27th and will try to work on ListingsItemService! I have a feeling REST api could be faster for price updates.

**Reply by abuzuhri** [reply]
> I will be free week of 27th and will try to work on ListingsItemService! I have a feeling REST api could be faster for price updates.
Hello @igoryan-k 
Greet , Please proceed and ask me if you want any help , And thanks in advance
